source/js/_module.js

```diff
@@ -0,0 +1,238 @@
+"use strict";
+console.log('Hello friend');
+let elements = {
+    phoneBox: document.getElementById('items'),
+    pagination: document.getElementById('pagination'),
+    search: document.getElementById('search__input'),
+    cartBtn: document.getElementById('header-cart'),
+    cartClose: document.getElementById('cart__close'),
+    cartWrapper: document.getElementById('cart__wrapper'),
+    cartDashboard: document.getElementById('dashboard'),
+    cartContent: document.getElementById('cart__content'),
+    bodyElement: document.body
+}
+let cartElementString = '';
+let cartTemplate = [];
+let template = ['','','','','','']; // если так не делать будет undefinde
+let phonesArr = [
```

Comment:
if you wont change data use const

Reply:
The data is changed in the addIdNumberAndFlag function, I add "idIndex" and "inCart"